client/src/pages/NeedHelp.js

```diff
@@ -153,8 +153,8 @@ const NeedHelp = withRouter((props) => {
 
   useEffect(() => {
     setTransition(!transition);
-  },[]);
-  
+  }, [transition]);
```

Comment:
Also need to update this one here. This will cause an infinite loop.

Reply:
Yeah, I messed it up with the ES linter.